docs/typescript/programming-guide.md

```diff
@@ -259,7 +251,7 @@ We recommend using transactions because:
 1. They give you access to a pre-configured database client, which is more convenient than connecting to the database yourself. DBOS integrates with most popular TypeScript ORMs, including Knex, Prisma, TypeORM, and Drizzle, and also supports raw SQL.
 2. Under the hood, transactions are highly optimized because DBOS can update its record of your program's execution _inside_ your transaction. For more info, see our ["how workflows work"](../explanations/how-workflows-work.md) explainer.
 
-Now, rebuild your app with with `npm run build`, restart with `npx dbos start`, and visit its URL again: http://localhost:3000/greeting/Mike.
+Now, rebuild your app with with `npm run build`, restart with `npm run start`, and visit its URL again: http://localhost:3000/greeting/Mike.
```

Comment:
Why not use `npx dbos start`?

Reply:
I like this more generic command, looks natural and familiar